lib/msal-node/src/crypto/CryptoOps.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ICrypto, PkceCodes } from '@azure/msal-common';
+import { GuidGenerator } from './GUidGenerator';
+import { Base64Encode } from './../encode/Base64Encode';
+import { Base64Decode } from './../encode/Base64Decode';
+import { PkceGenerator } from './PkceGenerator';
+
+/**
+ * This class implements MSAL node's crypto interface, which allows it to perform base64 encoding and decoding, generating cryptographically random GUIDs and
+ * implementing Proof Key for Code Exchange specs for the OAuth Authorization Code Flow using PKCE (rfc here: https://tools.ietf.org/html/rfc7636).
+ */
+export class CryptoOps implements ICrypto {
```

Comment:
What does the name `CryptoOps` mean?

Reply:
I would probably argue for a different name ("crypt-oops", Freudian slip?), but its consistent with MSAL browser: https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/1107/files#diff-33a6cab8b15c1d5ebe0810440a955ca9

Reply:
The naming, did not have a choice since the interface defined it already. I am fine with anything the team decides.

Reply:
Interface only defines function names, not class name. happy to rename, please provide some alternatives as CryptoUtils didn't sound appropriate.